### PR TITLE
 Merge dev-es into main (for the first Spanish localization live version)

### PR DESCRIPTION
**Pull request by electrocucaracha**
Merge Spanish development branch into main.

Requirements to go live have been completed.

    https://github.com/cncf/glossary/projects/5

Check list to make localization live.

- [x] validate all files which are not in the localization directory (content/es/*)
- [x] check i18n/es.toml exist
- [x] check l10n configuration in config.toml
- [x] acknowledge to initial localization contributors list
- [x] check Preview works

### CODE REVIEW

**Comment by seokho-son** [comment]
Hello, @electrocucaracha :)
Please add acknowledgement to Spanish localization contributors to [_index.md](https://github.com/cncf/glossary/pull/806/files#diff-a0053884521086aea9189a574a7ba5afc2e459da6e4c717382f9eb48cb60399aR21). (similar style with ack to Glossary contribution)

You can open another PR to add it.

**Comment by electrocucaracha** [comment]
> Hello, @electrocucaracha :) Please add acknowledgement to Spanish localization contributors to [_index.md](https://github.com/cncf/glossary/pull/806/files#diff-a0053884521086aea9189a574a7ba5afc2e459da6e4c717382f9eb48cb60399aR21). (similar style with ack to Glossary contribution)
> 
> You can open another PR to add it.

Ok, I wasn't sure about those changes. I opened the #807 PR

**Review by krol3 — APPROVED** [review]
LGTM

**Comment by jihoon-seo** [comment]
@electrocucaracha In preview, the Spanish version of ["Contributor Ladder" page](https://deploy-preview-806--cncfglossary.netlify.app/contributor-ladder/) is missing.
Could you (or anyone in Spanish L10n team) add this? 😊

File: content/es/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+# Glosario Cloud Native
+
+El Glosario Cloud Native es un proyecto liderado por la CNCF Business Value Subcommittee (BVS). Su meta es explicar los conceptos nativos para la nube en un lenguaje claro y simple sin recurrir a cualquier conocimiento técnico previo.
+
+## Contribuyendo
+Todo mundo esta invitado a sugerir cambios, adiciones, y mejoras a el Glosario Cloud Native. Empleamos un proceso guiado por la comunidad y gobernado por la CNCF para desarrollar y mejorar este léxico compartido. Este Glosario provee una plataforma neutral de vendedores para organizar un vocabulario compartido alrededor de las tecnologías nativas para la nube. Las contribuciones son bienvenidas de todos los participantes que cumplan con el propósito y los estatutos del proyecto.
+
+Cualquier persona que desee hacer una contribución puede enviar un GitHub issue o crear un pull request. Asegúrese de seguir la [Guía de estilo](/style-guide/), leer el documento de [Cómo contribuir](/contribute/) y unirse al canal de #glossary en el CNCF Slack. También hay un canal #glossary-localizations para aquellos que quieran ayudar a traducir el glosario a su idioma.
```
Comment (jihoon-seo) [inline content/es/_index.md:13]: Let's link to the Spanish versions!

```suggestion
Cualquier persona que desee hacer una contribución puede enviar un GitHub issue o crear un pull request. Asegúrese de seguir la [Guía de estilo](/es/style-guide/), leer el documento de [Cómo contribuir](/es/contribute/) y unirse al canal de #glossary en el CNCF Slack. También hay un canal #glossary-localizations para aquellos que quieran ayudar a traducir el glosario a su idioma.
```

File: content/es/_index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+title: "Glosario Cloud Native"
+---
+
+# Glosario Cloud Native
+
+El Glosario Cloud Native es un proyecto liderado por la CNCF Business Value Subcommittee (BVS). Su meta es explicar los conceptos nativos para la nube en un lenguaje claro y simple sin recurrir a cualquier conocimiento técnico previo.
+
+## Contribuyendo
+Todo mundo esta invitado a sugerir cambios, adiciones, y mejoras a el Glosario Cloud Native. Empleamos un proceso guiado por la comunidad y gobernado por la CNCF para desarrollar y mejorar este léxico compartido. Este Glosario provee una plataforma neutral de vendedores para organizar un vocabulario compartido alrededor de las tecnologías nativas para la nube. Las contribuciones son bienvenidas de todos los participantes que cumplan con el propósito y los estatutos del proyecto.
```
Comment (jihoon-seo) [inline content/es/_index.md:11]: I suggest to insert empty lines before and after the header lines. 😊

- https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines
- https://arcticicestudio.github.io/styleguide-markdown/rules/headings.html#empty-lines-before-and-after

```suggestion
## Contribuyendo

Todo mundo esta invitado a sugerir cambios, adiciones, y mejoras a el Glosario Cloud Native. Empleamos un proceso guiado por la comunidad y gobernado por la CNCF para desarrollar y mejorar este léxico compartido. Este Glosario provee una plataforma neutral de vendedores para organizar un vocabulario compartido alrededor de las tecnologías nativas para la nube. Las contribuciones son bienvenidas de todos los participantes que cumplan con el propósito y los estatutos del proyecto.
```

File: content/es/abstraction.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: Abstracción
+status: Completed
+category: Propiedad
+---
+
+En el contexto de la informática, una abstracción es una representación que oculta los detalles específicos a un consumidor de [servicios](https://glossary.cncf.io/service/) (un consumidor que es un programa de computadora o un ser humano), haciendo un sistema más genérico y, por lo tanto, más fácil de entender. Un buen ejemplo es el sistema operativo (SO) de su computadora portátil. No necesita saber nada sobre el CPU, la memoria y cómo se manejan los programas, solo necesita operar el sistema operativo y el sistema operativo se ocupa de los detalles. Todos estos detalles se encuentran ocultos detrás de la "cortina" del sistema operativo o la abstracción.
```
Comment (jihoon-seo) [inline content/es/abstraction.md:7]: I suggest to remove hardcoded site domain name:

```suggestion
En el contexto de la informática, una abstracción es una representación que oculta los detalles específicos a un consumidor de [servicios](/service/) (un consumidor que es un programa de computadora o un ser humano), haciendo un sistema más genérico y, por lo tanto, más fácil de entender. Un buen ejemplo es el sistema operativo (SO) de su computadora portátil. No necesita saber nada sobre el CPU, la memoria y cómo se manejan los programas, solo necesita operar el sistema operativo y el sistema operativo se ocupa de los detalles. Todos estos detalles se encuentran ocultos detrás de la "cortina" del sistema operativo o la abstracción.
```

File: content/es/api_gateway.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: API Gateway
+status: Completed
+category: Tecnología
+---
+
+## ¿Qué es esto?
+Un [API Gateway](/application_programming_interface/) es una herramienta que concentra diferentes aplicaciones APIs, centralizando a todas en un solo lugar. Esto permite a las organizaciones mover funciones clave, como puede ser la autenticación y la autorización o limitando el número de peticiones entre aplicaciones a una ubicación de administración centralizada. Un API Gateway funciona como una interface común para los que la consumen (usualmente aplicaciones externas).
```
Comment (jihoon-seo) [inline content/es/api_gateway.md:8]:
```suggestion
Un [API](/application_programming_interface/) Gateway es una herramienta que concentra diferentes aplicaciones APIs, centralizando a todas en un solo lugar. Esto permite a las organizaciones mover funciones clave, como puede ser la autenticación y la autorización o limitando el número de peticiones entre aplicaciones a una ubicación de administración centralizada. Un API Gateway funciona como una interface común para los que la consumen (usualmente aplicaciones externas).
```

File: content/es/contribute/_index.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
+---
+title: Como contribuir
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-pen-square'></i>
```
Comment (jihoon-seo) [inline content/es/contribute/_index.md:7]: Let's remove topnav icons:

```suggestion
```

File: content/es/style-guide/_index.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+**Ejemplo**: eche un vistazo a la sección "Qué es" de la [definición de service mesh](https://glossary.cncf.io/service_mesh/). Se vincula con las definiciones de microservicios, servicio, confiabilidad y observabilidad. Además, utiliza un ejemplo del mundo real que compara los desafíos de la red en un entorno de microservicios (algo con lo que las personas no técnicas no pueden relacionarse) con los problemas de wifi (algo que cualquiera que use una computadora portátil puede entender). Siempre que sea posible, trate de hacer esa conexión.
+
+#### Comience con un documento de Google o Word
+
+Recomendamos comenzar con un documento de Google o Word, dejalo pasar durante unos días y volver a visitarlo. Esto te permitirá captar frases o expresiones que podrían redactarse de una forma más sencilla y accesible. Además, asegúrese de ejecutar un corrector ortográfico antes de enviar un PR.
+
+Para asegurarse de que nadie más envíe un PR mientras trabaja en un término, asegúrese de reclamar un issue (o crear uno) y que se le asigne. Más sobre eso en el documento [Cómo contribuir](https://glossary.cncf.io/contribute/).
```
Comment (jihoon-seo) [inline content/es/style-guide/_index.md:122]:
```suggestion
Para asegurarse de que nadie más envíe un PR mientras trabaja en un término, asegúrese de reclamar un issue (o crear uno) y que se le asigne. Más sobre eso en el documento [Cómo contribuir](/es/contribute/).
```

File: content/es/style-guide/_index.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+Antes de comenzar, lea algunos de los términos publicados del Glosario para tener una idea del nivel de detalle y dificultad y cuándo los ejemplos son apropiados.
+
+
+## El proceso de revisión: qué esperar
+
+Tenga en cuenta que actualmente solo hay tres mantenedores haciendo esto en su tiempo libre. Ocasionalmente, podremos revisar los términos rápidamente; en otras ocasiones, puede llevar algo de tiempo. Agradecemos su paciencia. Si tiene alguna pregunta, comuníquese con nosotros en el canal #glossary de Slack (para saber dónde y cómo encontrarlo, consulte nuestro documento de [Cómo contribuir](https://glossary.cncf.io/contribute/)).
```
Comment (jihoon-seo) [inline content/es/style-guide/_index.md:129]:
```suggestion
Tenga en cuenta que actualmente solo hay tres mantenedores haciendo esto en su tiempo libre. Ocasionalmente, podremos revisar los términos rápidamente; en otras ocasiones, puede llevar algo de tiempo. Agradecemos su paciencia. Si tiene alguna pregunta, comuníquese con nosotros en el canal #glossary de Slack (para saber dónde y cómo encontrarlo, consulte nuestro documento de [Cómo contribuir](/es/contribute/)).
```

File: content/es/contribute/_index.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+3) Ayudar a traducir el glosario
+
+## ¡Únete a la comunidad del glosario!
+Considere unirse a nuestras reuniones mensuales del Grupo de trabajo del glosario si es que desea contribuir con regularidad. Puede encontrar los detalles de la reunión en el calendario de la CNCF. También puede conectarse con los mantenedores y otros colaboradores en nuestro canal #glossary en el [calendario CNCF](https://www.cncf.io/calendar/). ¡Nos encantaría conocerlo!
+
+## Proponer nuevos términos
+Puede proponer un nuevo término para que otros trabajen en él o crear una nueva definición usted mismo. De cualquier forma, comenzará creando un issue (tenga en cuenta que necesitará una cuenta de GitHub para hacerlo).
+
+A continuación se muestra una guía paso a paso para aquellos que aún no están familiarizados con GitHub. **Si eres un profesional de GitHub**, *da* un vistazo rápido para asegurarte de que usas nuestras plantillas de issues, las convenciones de nomenclatura, solicita un PR en Slack (de lo contrario, podemos perderlo) y dónde encontrar la plantilla del archivo. Y asegúrese de leer la [Guía de estilo](https://glossary.cncf.io/style-guide/) antes de comenzar. ¡Gracias!
```
Comment (jihoon-seo) [inline content/es/contribute/_index.md:24]:
```suggestion
A continuación se muestra una guía paso a paso para aquellos que aún no están familiarizados con GitHub. **Si eres un profesional de GitHub**, *da* un vistazo rápido para asegurarte de que usas nuestras plantillas de issues, las convenciones de nomenclatura, solicita un PR en Slack (de lo contrario, podemos perderlo) y dónde encontrar la plantilla del archivo. Y asegúrese de leer la [Guía de estilo](/es/style-guide/) antes de comenzar. ¡Gracias!
```

File: content/es/contribute/_index.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+Aquí puede ver que los primeros tres términos están disponibles mientras que el siguiente término ha sido asignado a alguien.
+
+![assigning a term](/images/how-to/howto-04.png)
+
+Tenga en cuenta que solo puede reclamar un término a la vez. Si desea trabajar en varios términos, termine uno antes de reclamar el siguiente.
+
+### Envío de un nuevo término (creando un PR)
+
+Antes de comenzar, lea la [Guía de estilo](https://glossary.cncf.io/style-guide/) — le ayudará a minimizar el ir de un lado a otro. Como se indica en la guía de estilo, recomendamos encarecidamente el comenzar con un documento de Google o Word.
```
Comment (jihoon-seo) [inline content/es/contribute/_index.md:49]:
```suggestion
Antes de comenzar, lea la [Guía de estilo](/es/style-guide/) — le ayudará a minimizar el ir de un lado a otro. Como se indica en la guía de estilo, recomendamos encarecidamente el comenzar con un documento de Google o Word.
```

File: content/es/style-guide/_index.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+Cuando corresponda, use **ejemplos del mundo real** que ayuden a los lectores (especialmente a los no técnicos) a comprender mejor *cuándo* y *por qué* el concepto que está explicando es relevante.
+
+Cuando se use en su definición, siempre **enlace a los términos existentes del glosario** (solo la primera mención debe tener un hipervínculo).
+
+**Ejemplo**: eche un vistazo a la sección "Qué es" de la [definición de service mesh](https://glossary.cncf.io/service_mesh/). Se vincula con las definiciones de microservicios, servicio, confiabilidad y observabilidad. Además, utiliza un ejemplo del mundo real que compara los desafíos de la red en un entorno de microservicios (algo con lo que las personas no técnicas no pueden relacionarse) con los problemas de wifi (algo que cualquiera que use una computadora portátil puede entender). Siempre que sea posible, trate de hacer esa conexión.
```
Comment (jihoon-seo) [inline content/es/style-guide/_index.md:116]:
```suggestion
**Ejemplo**: eche un vistazo a la sección "Qué es" de la [definición de service mesh](/service_mesh/). Se vincula con las definiciones de microservicios, servicio, confiabilidad y observabilidad. Además, utiliza un ejemplo del mundo real que compara los desafíos de la red en un entorno de microservicios (algo con lo que las personas no técnicas no pueden relacionarse) con los problemas de wifi (algo que cualquiera que use una computadora portátil puede entender). Siempre que sea posible, trate de hacer esa conexión.
```

File: content/es/style-guide/_index.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
+---
+title: Guia de estilos
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-ruler-horizontal'></i>
```
Comment (jihoon-seo) [inline content/es/style-guide/_index.md:7]: Let's remove topnav icons:

```suggestion
```

File: content/es/style-guide/_index.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+Esta guia de estilos te ayudará a entender la audiencia del Glosario, la estructura de las definiciones, el nivel de detalle requerido, y como mantener un estilo consistente.
+
+El Glosario Cloud Native sigue las [guias de estilo por defecto](https://github.com/cncf/foundation/blob/master/style-guide.md) del repositorio de CNCF. Adicionalmente, sigue las siguientes reglas:
+
+1. Utilice un lenguaje sencillo y accesible, evitando la jerga técnica y palabras de moda.
+2. [Evite el lenguaje coloquial](https://en.wikipedia.org/wiki/Colloquialism)
+3. [Use un lenguaje literal y concreto](http://guidetogrammar.org/grammar/composition/abstract.htm)
```
Comment (jihoon-seo) [inline content/es/style-guide/_index.md:16]:
```suggestion
3. [Use un lenguaje literal y concreto](https://guidetogrammar.org/grammar/composition/abstract.htm)
```

File: content/es/style-guide/_index.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+4. [Omite contracciones](https://en.wikipedia.org/wiki/Contraction_(grammar))
+5. [Use la voz pasiva con moderación] (https://www.ef.com/ca/english-resources/english-grammar/passive-voice/)
+6. [Trate de formular declaraciones en forma positiva] (https://examples.yourdictionary.com/positive-sentence-examples.html)
+7. [Sin signos de exclamación fuera de las comillas] (https://www.grammarly.com/blog/exclamation-mark/)
```
Comment (jihoon-seo) [inline content/es/style-guide/_index.md:20]: To make links work:

```suggestion
5. [Use la voz pasiva con moderación](https://www.ef.com/ca/english-resources/english-grammar/passive-voice/)
6. [Trate de formular declaraciones en forma positiva](https://examples.yourdictionary.com/positive-sentence-examples.html)
7. [Sin signos de exclamación fuera de las comillas](https://www.grammarly.com/blog/exclamation-mark/)
```

File: content/es/contribute/_index.md [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+
+### Actualizar un término directamente
+Para cambiar un término directamente, vaya a "Editar esta página".
+
+![edit this page](/images/how-to/howto-16.png)
+
+Esto abrirá la página de término en GitHub. Realice sus cambios y envíe un PR. Consulte "Envío de un nuevo término" más arriba para obtener una descripción mas detallada (salte a la sección que habla acerca de markdown).
+
+## Ayuda a traducir el glosario
+Para ayudar a traducir el glosario a su idioma, únase al canal #glossary-localizations en CNCF Slack y háganoslo saber. Puede unirse a un equipo existente o crear uno nuevo (para ver lo que se necesita, consulte la [Guía de localización](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md)). Únase también a nuestras juntas mensuales del grupo de trabajo. Puede encontrar mas detalles en el [calendario de la CNCF]((https://www.cncf.io/calendar/). ¡Esperamos verte allí!
```
Comment (jihoon-seo) [inline content/es/contribute/_index.md:107]:
```suggestion
Para ayudar a traducir el glosario a su idioma, únase al canal #glossary-localizations en CNCF Slack y háganoslo saber. Puede unirse a un equipo existente o crear uno nuevo (para ver lo que se necesita, consulte la [Guía de localización](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md)). Únase también a nuestras juntas mensuales del grupo de trabajo. Puede encontrar mas detalles en el [calendario de la CNCF](https://www.cncf.io/calendar/). ¡Esperamos verte allí!
```

**Review by jihoon-seo — COMMENTED** [review]
Dear Spanish L10n team,
muchas gracias for your work!

And I left some comments. PTAL! 😊

File: content/es/_index.md (hunk 1 above)
Comment (electrocucaracha) [reply content/es/_index.md:13]: Fix on 83a25403382163ec348343e4442ac4a4ccf86c60

File: content/es/_index.md (hunk 2 above)
Comment (electrocucaracha) [reply content/es/_index.md:11]: Fix on 4190b0989dcd8d1684da424859323f1da1c9abf2

File: content/es/style-guide/_index.md (hunk 13 above)
Comment (electrocucaracha) [reply content/es/style-guide/_index.md:20]: fix on 4190b0989dcd8d1684da424859323f1da1c9abf2

File: content/es/contribute/_index.md (hunk 14 above)
Comment (electrocucaracha) [reply content/es/contribute/_index.md:107]: Fix on 4190b0989dcd8d1684da424859323f1da1c9abf2

File: content/es/style-guide/_index.md (hunk 12 above)
Comment (electrocucaracha) [reply content/es/style-guide/_index.md:16]: Fix on bda4a052ed1ccdbed6a98fe89b74386225fb74d9

File: content/es/style-guide/_index.md (hunk 11 above)
Comment (electrocucaracha) [reply content/es/style-guide/_index.md:7]: Fix on b1fdd3def6f62d61e73f1436320f06f491b03479

File: content/es/style-guide/_index.md (hunk 10 above)
Comment (electrocucaracha) [reply content/es/style-guide/_index.md:116]: Fix on e82e13c76b683cee9be5f5b7fa8816d5c9919522

File: content/es/abstraction.md (hunk 3 above)
Comment (electrocucaracha) [reply content/es/abstraction.md:7]: Fix on e82e13c76b683cee9be5f5b7fa8816d5c9919522

File: content/es/api_gateway.md (hunk 4 above)
Comment (electrocucaracha) [reply content/es/api_gateway.md:8]: Fix on e103561e96c1694cfd0929314da4982deda57e8f

**Comment by seokho-son** [comment]
This PR looks good to me except the issue @jihoon-seo commented.

> @electrocucaracha In preview, the Spanish version of ["Contributor Ladder" page](https://deploy-preview-806--cncfglossary.netlify.app/contributor-ladder/) is missing. Could you (or anyone in Spanish L10n team) add this? 😊

But, I think it is ok without Spanish version of "Contributor Ladder" page for the first live version. 
The page can be included in a next round.

I think decision is up to Spanish team :)

@electrocucaracha do you want include Spanish version of "Contributor Ladder" this time ?

**Comment by electrocucaracha** [comment]
> This PR looks good to me except the issue @jihoon-seo commented.
> 
> > @electrocucaracha In preview, the Spanish version of ["Contributor Ladder" page](https://deploy-preview-806--cncfglossary.netlify.app/contributor-ladder/) is missing. Could you (or anyone in Spanish L10n team) add this? 😊
> 
> But, I think it is ok without Spanish version of "Contributor Ladder" page for the first live version. The page can be included in a next round.
> 
> I think decision is up to Spanish team :)
> 
> @electrocucaracha do you want include Spanish version of "Contributor Ladder" this time ?

@seokho-son I'm working on that document, but it may require some time to give eyes from reviewers. If this PR doesn't brake anything of `main` I'd suggest proceeding to merge it.

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Review by seokho-son — APPROVED** [review]
LGGGGGGGTM ! :)

**Pull request merged**